wiki/models/__init__.py

```diff
@@ -72,6 +89,23 @@ def reverse(*args, **kwargs):
         url = original_django_reverse(*args, **kwargs)
     
     return url
+
     
 # Now we redefine reverse method
-urlresolvers.reverse = reverse
\ No newline at end of file
+reverse_lazy = lazy(reverse, text_type)
```

Comment:
Redefining reverse_lazy caused me some grief initially because I wasn't using text_type, and instead using basestring. When I switched to text_type (which is what is currently in django-wiki upstream), the problem was resolved.

Note that this implementation of reverse_lazy is consistent with the default implementation, except that it will go through their custom reverse method above.